explorer/files/ncpi-catalog/ncpi-update-utils.ts

```diff
@@ -0,0 +1,25 @@
+import fs from "fs";
+import { dbGapId } from "./entities";
+
+export const sourcePath = "ncpi-catalog/files/dashboard-source-ncpi.tsv";
+
+export function appendToTsv(path: string, data: string[][]): void {
+  let out = "";
```

Comment:
Hi @jpaten can you convert this to use the tsv/csv library to write out the file? this way, the library can properly do the escaping say of data with tabs in it. I think there should be some examples in the AnVIL repo. If not look at the library docs there should be examples of how to do it.